apps/member-profile/app/routes/_profile.home.claim-swag-pack._index.tsx

```diff
@@ -164,15 +167,10 @@ function ClaimSwagPackForm() {
         </Form.Field>
 
         <Address.HalfGrid>
-          <Form.Field
-            error={errors.addressCity}
-            label="City"
-            labelFor={keys.addressCity}
-            required
-          >
-            <Address.City
-              id={keys.addressCity}
-              name={keys.addressCity}
+          <Form.Field label="Country" labelFor={keys.addressCountry} required>
+            <Address.Country
+              id={keys.addressCountry}
+              name={keys.addressCountry}
```

Comment:
Let's move the country part of the form to after the zipcode field in this file. It makes more sense to have the ordering with `addressLine1`, `addressLine2`, `addressCity`, ..., `addressCountry` rather than having `addressCountry` after `addressLine2`.